### PR TITLE
Cinnamon Maple Whiskey Sour drink

### PR DESCRIPTION
**Pull request by jjohn50**
The manager of the repo requested I made changes to the tags and image. I fixed them.

### CODE REVIEW

**Review by alfg — CHANGES_REQUESTED** [review]
The JSON and image filename should match, using hyphens instead of the underscores.
```
cinnamon-maple-whiskey-sour.jpg
cinnamon-maple-whiskey-sour.json
```

File: src/recipes/cinnamon_maple_whiskey.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "name": "Cinnamon Maple Whiskey Sour>",
```
Comment (alfg) [inline src/recipes/cinnamon_maple_whiskey.json:2]:
```suggestion
    "name": "Cinnamon Maple Whiskey Sour",
```

**Review by alfg — CHANGES_REQUESTED** [review]

File: src/recipes/cinnamon_maple_whiskey.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    ],
+    "directions": [
+        "Fill a cocktail shaker or mason jar about two-thirds full with ice. ",
+        "Pour in the bourbon, lemon juice, maple syrup and a pinch of ground cinnamon. ",
+        "Securely fasten the lid and shake well.",
+        "Fill a cocktail shaker or mason jar about two-thirds full with ice. ",
+        "Pour fresh ice into your cocktail glass and strain the cold whiskey sour mixture into the glass. Enjoy!"
+    ],
+    "image": "cinnamon-maple-whiskey-sour.jpg",
+    "source": "https://cookieandkate.com/images/2015/12/cinnamon-maple-whiskey-sour-3-768x1056.jpg",
```
Comment (alfg) [inline src/recipes/cinnamon_maple_whiskey.json:35]: Link to the recipe here, instead of the image.

Comment (jjohn50) [reply src/recipes/cinnamon_maple_whiskey.json:35]: The link is the link to the recipe from the website I found it from.

Comment (alfg) [reply src/recipes/cinnamon_maple_whiskey.json:35]: Ok, you can remove this line if there's no source page.

**Comment by jjohn50** [comment]
I'm pretty sure I fixed the changes request, but let me know.

**Comment by alfg** [comment]
Thanks!

**Pull request merged**